packages/provider-elasticsearch/src/example-types/metricGroups/pivot.js

```diff
@@ -292,20 +292,44 @@ let createPivotScope = (node, schema, getStats) => {
         let build = lookupTypeProp(_.identity, 'buildGroupQuery', group.type)
         //Remove anything that needs to be hoisted before composing further
         // We are iterating through the groups reversed so we need to subtract instead of add to get the right index
-        let reversedLookupIndex = groups.length - index - 1
+        let groupIndex = groups.length - index - 1
         let drilldownKey = _.get(
-          [groupingType, 'drilldown', reversedLookupIndex],
+          [groupingType, 'drilldown', groupIndex],
           request
         )
 
-        let parent = await build(
-          group,
-          await children,
-          groupingType,
-          schema,
-          getStats,
-          drilldownKey
-        )
+        let withGroupingCardinality =
+          _.get('expanded.cardinality', node) &&
```

Comment:
Changes:

- Move `groupCounts: true | false` to a per grouping config value
- Add `skip: true | false` per grouping

This will enable UI features to show paging info within the UI